patches/chrome-installer-mini_installer-mini_installer.cc.patch

```diff
@@ -116,22 +116,26 @@ index 1f39abbeb2579098a54fa82612199f0307ea03b4..6aa317767ed09e31296c5c9af187481a
 +}
 +
 +bool GetStubInstallerFilename(PathString& installer_filename) {
-+  wchar_t value[MAX_PATH];
-+  if (RegKey::ReadSZValue(HKEY_CURRENT_USER, L"Software\\BraveSoftware\\Promo",
-+                          L"StubInstallerPath", value, _countof(value)) &&
-+      value[0] == L'0') {
++  wchar_t value[MAX_PATH] = {0,};
++  const bool result =
++      RegKey::ReadSZValue(HKEY_CURRENT_USER, L"Software\\BraveSoftware\\Promo",
++                          L"StubInstallerPath", value, _countof(value)) ;
++  if (result && value[0] == L'0')
```

Comment:
Should we compare result against `ERROR_SUCCESS` just to be completely explicit?

Reply:
I found that when `value` that read is empty, install was still failed.
So, checking emptiness of it is checked when when success.